public/app/features/variables/query/operators.ts

```diff
@@ -70,7 +70,7 @@ export function toMetricFindValues(): OperatorFunction<PanelData, MetricFindValu
           throw new Error("Couldn't find any field of type string in the results.");
         }
 
-        for (const frame of frames) {
+        for (const frame of processedDataFrames) {
```

Comment:
The `frames` may not be dataFrames at all, we process the response earlier, find fields in the processed dataFrames but then use the original data structures here which does not make sense.